igneous/tasks/mesh/multires.py

```diff
@@ -531,8 +531,54 @@ def create_octree_level_from_mesh(mesh, chunk_shape, lod, num_lods):
         if np.sum([ np.all(mesh_z.vertices[:,i] == mesh_z.vertices[0,i]) for i in range(3) ]) >= 2:
           continue
 
-        submeshes.append(mesh_z)
-        nodes.append((x, y, z))
+        yield mesh_z, (x, y, z)
+
+def retriangulate_mesh(
+    mesh: trimesh.Trimesh,
+    offset: np.ndarray,
+    grid_size: Vec,
+    scale: Vec,
+  ) -> trimesh.Trimesh:
+  """
+  Retriangulate the input mesh to avoid any cases where the boundaries of a triangle are split across the boundaries of the submeshes
+  """
+  new_mesh = trimesh.Trimesh()
+
+  for submesh, _ in generate_gridded_submeshes(
+      mesh, offset, grid_size, scale):
```

Comment:
Annoying style point, but `):` should be on the next line (consistent with below modification too).